server/routes/visitJourney/dateAndTime.test.ts

```diff
@@ -34,117 +34,6 @@ const testJourneys = [
-    {
-      date: 'Monday 14 February',
-      prisonerEvents: {
-        morning: [],
-        afternoon: [],
-      },
-      slots: {
-        morning: [
-          {
-            id: '1',
-            startTimestamp: '2022-02-14T10:00:00',
-            endTimestamp: '2022-02-14T11:00:00',
-            availableTables: 15,
-            capacity: 30,
-            visitRoomName: 'room name',
-            // representing a pre-existing visit that is BOOKED
-            sessionConflicts: ['DOUBLE_BOOKED'],
-            visitRestriction: 'OPEN',
-          },
-          {
-            id: '2',
-            startTimestamp: '2022-02-14T11:59:00',
-            endTimestamp: '2022-02-14T12:59:00',
-            availableTables: 1,
-            capacity: 30,
-            visitRoomName: 'room name',
-            visitRestriction: 'OPEN',
-          },
-        ],
-        afternoon: [
-          {
-            id: '3',
-            startTimestamp: '2022-02-14T12:00:00',
-            endTimestamp: '2022-02-14T13:05:00',
-            availableTables: 5,
-            capacity: 30,
-            visitRoomName: 'room name',
-            // representing the RESERVED visit being handled in this session
-            sessionConflicts: ['DOUBLE_BOOKED'],
-            visitRestriction: 'OPEN',
-          },
-        ],
-      },
-    },
-    {
-      date: 'Tuesday 15 February',
-      prisonerEvents: {
-        morning: [],
-        afternoon: [],
-      },
-      slots: {
-        morning: [],
-        afternoon: [
-          {
-            id: '4',
-            startTimestamp: '2022-02-15T16:00:00',
-            endTimestamp: '2022-02-15T17:00:00',
-            availableTables: 12,
-            capacity: 30,
-            visitRoomName: 'room name',
-            visitRestriction: 'OPEN',
-          },
-          {
-            id: '5',
-            startTimestamp: '2022-02-15T16:00:00',
-            endTimestamp: '2022-02-15T17:00:00',
-            availableTables: 12,
-            capacity: 30,
-            visitRoomName: 'room name',
-            visitRestriction: 'CLOSED',
-          },
-        ],
-      },
-    },
-  ],
-  'March 2022': [
-    {
-      date: 'Tuesday 1 March',
-      prisonerEvents: {
-        morning: [],
-        afternoon: [],
-      },
-      slots: {
-        morning: [
-          {
-            id: '6',
-            startTimestamp: '2022-03-01T09:30:00',
-            endTimestamp: '2022-03-01T10:30:00',
-            availableTables: 0,
-            capacity: 30,
-            visitRoomName: 'room name',
-            visitRestriction: 'OPEN',
-          },
-          {
-            id: '7',
-            startTimestamp: '2022-03-01T09:30:00',
-            endTimestamp: '2022-03-01T10:30:00',
-            availableTables: 0,
-            capacity: 30,
-            visitRoomName: 'room name',
-            visitRestriction: 'CLOSED',
-          },
-        ],
-        afternoon: [],
-      },
-    },
-  ],
-}
-
 beforeEach(() => {
```

Comment:
Git diff making changes to this file look more complicated they are - have basically changed scope of some test data and re-done a new block of tests at the end to cover restriction change scenarios.